src/test/resources/release-notes-file.md

```diff
@@ -0,0 +1,11 @@
+### New features
+
+* [JIRA-TICKET](jira-link): Some feature
```

Comment:
As we discuss, an example of entire project history on one place:
https://github.com/Ontotext-AD/release-test/releases